### PR TITLE
Publish WorkItems as Microsoft.WorkItems

### PR DESCRIPTION
**Pull request by eddynaka**

### CODE REVIEW

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Review by michaelcfanning — CHANGES_REQUESTED** [review]
:clock1:

File: src/WorkItems/WorkItems.csproj [hunk 1]
```diff
@@ -4,6 +4,14 @@
     <RootNamespace>Microsoft.WorkItems</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Label="Packaging">
+    <!--
+    Without this property setting, the id of the NuGet package would match the
+    assembly name, "WorkItems".
```
Comment (michaelcfanning) [inline src/WorkItems/WorkItems.csproj:10]: Let's actually change the name of the assembly to Microsoft.WorkItems. :)

Comment (eddynaka) [reply src/WorkItems/WorkItems.csproj]: just updated! :)

**Review by michaelcfanning — APPROVED** [review]
:shipit:

File: src/WorkItems/WorkItems.csproj (hunk 1 above)
Comment (michaelcfanning) [reply src/WorkItems/WorkItems.csproj]: Merge it! :)

---
In reply to: [542029000](https://github.com/microsoft/sarif-sdk/pull/2180#discussion_r542029000) [](ancestors = 542029000)

Comment (eddynaka) [reply src/WorkItems/WorkItems.csproj]: fixing the build and will merge it. thanks for reviewing it

**Pull request merged**